testsuite/MDAnalysisTests/analysis/test_pca.py

```diff
@@ -0,0 +1,48 @@
+# -*- Mode: python; tab-width: 4; indent-tabs-mode:nil; coding:utf-8 -*-
+# vim: tabstop=4 expandtab shiftwidth=4 softtabstop=4 fileencoding=utf-8
+#
+# MDAnalysis --- http://www.MDAnalysis.org
+# Copyright (c) 2006-2015 Naveen Michaud-Agrawal, Elizabeth J. Denning, Oliver
+# Beckstein and contributors (see AUTHORS for the full list)
+#
+# Released under the GNU Public Licence, v2 or any higher version
+#
+# Please cite your use of MDAnalysis in published work:
+#
+# N. Michaud-Agrawal, E. J. Denning, T. B. Woolf, and O. Beckstein.
+# MDAnalysis: A Toolkit for the Analysis of Molecular Dynamics Simulations.
+# J. Comput. Chem. 32 (2011), 2319--2327, doi:10.1002/jcc.21787
+#
+from __future__ import print_function
+import numpy as np
+import MDAnalysis
+import MDAnalysis.analysis.pca as pca
+from numpy.testing import (assert_almost_equal, assert_equal,
+                           assert_array_almost_equal,raises)
+
+
+from MDAnalysisTests.datafiles import PDB, XTC
+
+
```

Comment:
please test random-data for the cosine content as I have done for the examples I send you.